src/main/java/com/blackducksoftware/integration/hub/alert/Application.java

```diff
@@ -56,7 +56,7 @@
 import com.google.gson.GsonBuilder;
 
 @EnableAutoConfiguration(exclude = { BatchAutoConfiguration.class })
-@EnableJpaRepositories(basePackages = { "com.blackducksoftware.integration.hub.alert.datasource.repository" })
+@EnableJpaRepositories(basePackages = { "com.blackducksoftware.integration.hub.**.datasource.repository" })
```

Comment:
Is the plant to have multiple 'datasource.repository' packages? Why wouldnt we put the classes in the same package?

Reply:
We will be creating different *datasource.repository* packages in each channel, so we want to pick up each of those. I would be alright with switching to use the same package for all channels in this case because it's focused on DB operations and not channel operations. @psantos1113 requested separate packages, so let's discuss with him.